src/main/java/bio/terra/service/job/StairwayExceptionFieldsFactory.java

```diff
@@ -86,15 +96,19 @@ private static List<String> getStepExceptionDetails(Exception exception) {
   }
 
   private static List<String> getErrorReportExceptionCauses(Exception exception) {
-    if (ErrorReportException.class.isAssignableFrom(exception.getClass())) {
-      // Java 17 will make casting unnecessary
-      ErrorReportException errorReportException = (ErrorReportException) exception;
+    if (exception instanceof ErrorReportException errorReportException) {
+      List<String> causes = new ArrayList<>();
       if (!errorReportException.getCauses().isEmpty()) {
-        return errorReportException.getCauses();
+        causes.addAll(errorReportException.getCauses());
+      }
+      if (errorReportException.getCause() != null) {
+        causes.add(errorReportException.getCause().getMessage());
       }
+      causes.removeIf(StringUtils::isEmpty);
+      return causes;
     } else if (StringUtils.isNotEmpty(exception.getMessage())) {
       return List.of(exception.getMessage());
     }
-    return null;
+    return List.of();
```

Comment:
These changes surface the cause message. This should allow for users reading job result values to see the underlying exception in addition to the top level message - Fewer "Unknown exceptions; contact TDR team" messages.